lib/Supervisor.js

```diff
@@ -201,7 +201,23 @@ function run(
 
                 xml.testsuite.testcase = xml.testsuite.testcase.concat(values);
 
-                console.log(XmlBuilder.create(xml).end());
+                // The XmlBuilder by default does not remove characters that are
+                // invalid in XML, like backspaces. However, we can pass it an
+                // `invalidCharReplacement` option to tell it how to handle
+                // those characters, rather than crashing. In an attempt to
+                // retain useful information in the output, we try and output a
+                // hex-encoded unicode codepoint for the invalid character. For
+                // example, a backspace (`\u{08}` in Elm) will be output as
+                // `\0x8`.
```

Comment:
Is this format used in any language? If not, is there a reason to invent a new syntax rather than using some already existing one?